Boop/Boop/scripts/JoinLinesWithSpace.js

```diff
@@ -0,0 +1,15 @@
+/**
+	{
+		"api":1,
+		"name":"Join Lines With Space",
+		"description":"Joins All Lines With a Space",
+		"author":"riesentoaster",
+		"icon":"table",
+		"tags":"join, space",
+        "bias": -0.1
+	}
+**/
+
+function main(input) {
+	input.text = input.text.replace('\n', ',');
```

Comment:
Might be a good idea to actually replace with a space as specified. :-)
```suggestion
	input.text = input.text.replace('\n', ' ');
```